### ISSUE TITLE
Newlines and indentation not preserved for Jinja fragments

### ISSUE THREAD

**Issue by hyvyys** [davidodenwald/prettier-plugin-jinja-template#8]
This snippet:

```
{{ obj.method("argument")|default("", True) }}
{% with text='text', version=1 %}
    {% include 'directory/template.jinja2' %}
{% endwith %}
```

gets reformatted to this:

```
{{ obj.method("argument")|default("", True) }} {% with text='text', version=1 %}
{% include 'directory/template.jinja2' %}
{% endwith %}
```

Similarly, I find it problematic that newlines and indentation is not fully preserved (though I think setting `"htmlWhitespaceSensitivity": "strict"` helps preserve at least the base indentation within HTML):

```
{% with text='text', version=1 %}
    {% include 'directory/template.jinja2' %}
{% endwith %}

{% with text='text', version=1 %}
    {% include 'directory/template.jinja2' %}
{% endwith %}

<div>
    {% with text='text', version=1 %}
        {% include 'directory/template.jinja2' %}
    {% endwith %}
</div>
```

=>

```
{% with text='text', version=1 %}
{% include 'directory/template.jinja2' %}
{% endwith %} {% with text='text', version=1 %}
{% include 'directory/template.jinja2' %}
{% endwith %}

<div>
  {% with text='text', version=1 %}
  {% include 'directory/template.jinja2' %}
  {% endwith %}
</div>
```

**Comment by davidodenwald** [comment]
I didn't know about the `with`-statement.
It is added in the latest release 0.1.0.

**Comment by hyvyys** [comment]
Still doesn't work for me.

I got:

```
    "prettier": "^2.8.4",
    "prettier-plugin-jinja-template": "^0.1.0",
    "prettier-plugin-tailwindcss": "^0.2.4",
```

**Comment by hyvyys** [comment]
Btw I don't think you really need to be handling all jinja tags separately. After all, Jinja can be extended. Naively, I would just look for `{% %}`, `{{ }}` and `{#  #}` and leave them alone.

**Comment by davidodenwald** [comment]
Yes, this seems to be a better idea. Right now the plugin also checks if a opening statement is closed of by the correct closing keyword. But I guess this is not the duty of a formatter anyway.

Regarding my fix not working: My release pipeline broke and maybe the code wasn't released correctly.
Thanks for the feedback.

**Comment by davidodenwald** [comment]
@hyvyys Can you checkout the new release 0.1.1? It hopefully fixes the problems you where experiencing without breaking anything :crossed_fingers:

**Comment by Asim-Tahir** [comment]
**Input:**
```jinja
{% with text='text', version=1 %}
    {% include 'directory/template.jinja2' %}
{% endwith %}

{% with text='text', version=1 %}
    {% include 'directory/template.jinja2' %}
{% endwith %}

<div>
    {% with text='text', version=1 %}
        {% include 'directory/template.jinja2' %}
    {% endwith %}
</div>
```
>

**Output:**
>
```jinja
{% with text='text', version=1 %}
  {% include 'directory/template.jinja2' %}
{% endwith %}
{% with text='text', version=1 %}
  {% include 'directory/template.jinja2' %}
{% endwith %}

<div>
  {% with text='text', version=1 %}
    {% include 'directory/template.jinja2' %}
  {% endwith %}
</div>
```

**Expected Output:**
```jinja
{% with text='text', version=1 %}
  {% include 'directory/template.jinja2' %}
{% endwith %}

{% with text='text', version=1 %}
  {% include 'directory/template.jinja2' %}
{% endwith %}

<div>
  {% with text='text', version=1 %}
    {% include 'directory/template.jinja2' %}
  {% endwith %}
</div>
```

Hi @davidodenwald,
Not preserving whitespace issue that @hyvyys mentioned still exist at `v0.2.0`.

**Comment by davidodenwald** [comment]
Hey @Asim-Tahir, the whitespace issue should be fixed with the v0.4.0 release.
